http/document_service.go

```diff
@@ -58,6 +64,10 @@ func NewDocumentHandler(b *DocumentBackend) *DocumentHandler {
 	h.HandlerFunc("PUT", documentPath, h.handlePutDocument)
 	h.HandlerFunc("DELETE", documentPath, h.handleDeleteDocument)
 
+	h.HandlerFunc("GET", documentLabelsPath, h.handleGetDocumentLabel)
```

Comment:
Other services are using a function factory here (e.g. `newGetLabelsHandler`). Is it possible to use it here as well?